content/docs/concepts/replication/architecture/_index.md

```diff
@@ -124,8 +124,19 @@ remote protection groups on the backend storage array. This controller can also
 Both the PV & RG objects in the remote cluster have extra metadata associated with them in form of annotations & labels. This metadata includes
 information about the respective objects in the source cluster.
 
-The PVC objects are never replicated across the clusters. Instead, the remote PV objects have annotations related to the
-source PVC objects. This information can be easily used to create the PVCs whenever required using `repctl` or `kubectl`.
+`multi-cluster` The PVC objects are replicated across the clusters.
```

Comment:
Can you append to this sentence saying "when allowPvcCreationOnTarget=true", as by default allowPvcCreationOnTarget is false.

Reply:
done